.github/workflows/labels_from_comment.yml

```diff
@@ -1,4 +1,4 @@
-# This workflow allows any user to add one of the `awaiting-author`, or `WIP` labels,
+# This workflow allows any user to add or remove one of the `awaiting-author` or `WIP` labels,
 # by commenting on the PR or issue.
 # Other labels from this set are removed automatically at the same time.
```

Comment:
Still not 100% happy with the wording of my additions here, but this maybe makes things a bit more explicit.
```suggestion
# This workflow allows any user to add one of the `awaiting-author` or `WIP` labels,
# by commenting with the label name on the PR or issue. 
# The other label from this set is removed automatically at the same time.
# Commenting with `awaiting-review` removes both of these labels.
```

By the way, is it worth mentioning this workflow in our ["How to contribute to mathlib" page](https://leanprover-community.github.io/contribute/index.html#lifecycle-of-a-pr)?